src/main/java/Mona/MainWindow.java

```diff
@@ -0,0 +1,51 @@
+package mona;
+import javafx.fxml.FXML;
+import javafx.scene.control.Button;
+import javafx.scene.control.ScrollPane;
+import javafx.scene.control.TextField;
+import javafx.scene.image.Image;
+import javafx.scene.layout.AnchorPane;
+import javafx.scene.layout.VBox;
+/**
+ * Controller for MainWindow. Provides the layout for the other controls.
+ */
+public class MainWindow extends AnchorPane {
+    @FXML
+    private ScrollPane scrollPane;
+    @FXML
+    private VBox dialogContainer;
+    @FXML
+    private TextField userInput;
+    @FXML
+    private Button sendButton;
+
+    private Mona mona;
+
+    private Image userImage = new Image(this.getClass().getResourceAsStream("/images/Russer.jpeg"));
+    private Image dukeImage = new Image(this.getClass().getResourceAsStream("/images/Mona.jpeg"));
+
+    @FXML
+    public void initialize() {
+        scrollPane.vvalueProperty().bind(dialogContainer.heightProperty());
+    }
+
+    public void setDuke(Mona mona) {
+        this.mona = mona;
+    }
+
+    /**
+     * Creates two dialog boxes, one echoing user input and the other containing Duke's reply and then appends them to
+     * the dialog container. Clears the user input after processing.
+     */
+    @FXML
+    private void handleUserInput() {
```

Comment:
Good job with the naming of your methods, clear use of verbs and camelCase to adhere to the coding standards.